backend/core/models/user.py

```diff
@@ -194,11 +194,11 @@ class User(AbstractUser):
         validators=[validate_avatar],
         null=True,
     )
-    voting_right = models.FloatField(
+    trust_score = models.FloatField(
         null=True,
         blank=True,
         default=None,
-        help_text="The voting right assigned to the user based on the vouching mechanism.",
+        help_text="The trust score assigned to the user based on the vouching mechanism.",
```

Comment:
This would be more correct

```suggestion
        help_text="The trust score assigned to the user based on trusted domains and the vouching mechanism.",
```